README.md

```diff
@@ -24,6 +24,12 @@ To lock image references in a `Dockerfile`:
 locker Dockerfile
 ```
 
+To unlock image references in a `Dockerfile`:
+
+```bash
+locker Dockerfile --unlock
```

Comment:
You won't like this, but the convention is for switches to appear before arguments, i.e. `locker --unlock Dockerfile`. Fancy switching the switch? :)